CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 ### Add
 
 * Add integration test for `collection.count`, `find-cursor.count` and `find-cursor.sort`.
+* Add support for `mongodb@6.8.0` and up.
```

Comment:
I would say "the `mongodb@6.8.0` driver and up" to avoid any confusion with the version of mongodb server.